core/fee/rebate_stats.go

```diff
@@ -221,7 +191,7 @@ func (f *FeesStats) ToProto(asset string, assetQuantum num.Decimal) *eventspb.Fe
 		VolumeDiscountApplied:    make([]*eventspb.PartyAmount, 0, len(f.VolumeDiscountApplied)),
 		TotalMakerFeesReceived:   make([]*eventspb.PartyAmount, 0, len(f.TotalMakerFeesReceived)),
 		MakerFeesGenerated:       make([]*eventspb.MakerFeesGenerated, 0, len(f.MakerFeesGenerated)),
-		TradingFeesGenerated:     make([]*eventspb.TradingFeesGenerated, 0, len(f.TradingFeesGenerated)),
+		TotalFeesPaidAndReceived: make([]*eventspb.PartyAmount, 0, len(f.TradingFeesPaidAndReceived)),
```

Comment:
Doesn't this need to be added to the database in data node and the fee stats APIs?

Reply:
Confirmed with @karlem that it's not needed in DN